### DISCUSSION TITLE
Changelog?

### DISCUSSION THREAD

**Discussion by fdeross** [meisnate12/Plex-Meta-Manager#338] [Q&A] [answered]
Is there a changelog?  I've seen some updates come down but I'm not sure where to look to see bug fixes and new features.

**Comment by meisnate12** [comment, marked as answer]
this is my change log 

https://github.com/meisnate12/Plex-Meta-Manager/releases

changes in develop are not documented until they're released in case they change most times if there is a linked issue i usually put a description there of the change until its released

**Reply by fdeross** [reply]
Thank you for your hardwork!

**Reply by meisnate12** [reply]
no problem!
